dask/distributed/worker.py

```diff
@@ -530,6 +530,11 @@ def close(self):
     def __del__(self):
         self.close()
 
+    def __repr__(self):
+        s = '<dask.distributed.Worker at %s address=%s, scheduler=%s>'
+        return s % (hex(id(self)), self.address.decode('utf-8')[6:],
+                    self.scheduler.decode('utf-8')[6:])
```

Comment:
Oops, yes you are correct. I was misremembering a conversation we had about this and adding one worker per processor on anaconda cluster.